src/pages/Dashboard/TeamOverview.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { useQuery, gql } from '@apollo/client';
+import { Box, Button, Center, Flex, Heading, Spacer } from '@chakra-ui/react';
+
+const GET_TEAM_INFO = gql`
+  query getTeam($id: String!) {
+    getTeam(id: $id) {
+      name
+      organization
+      id
+      amountRaised
+      itemsSold
+    }
+  }
+`;
+
+const StorefrontButton = () => {
+  return <Button w="199px">Share Storefront</Button>;
+};
+
+const SalesInfo = props => {
+  return (
+    <Box mr="100px">
+      <Heading textTransform="uppercase" as="subtitle" size="subtitle" color="gray.500" mb="8px">
```

Comment:
heh so I don't think `subtitle` is a real HTMl tag. I think it would be semantically appropriate to render just a `p` tag here or an `h4` or `h5`? whatever you think is the most "semantic" lolol

```suggestion
      <Heading textTransform="uppercase" as="p" size="subtitle" color="gray.500" mb="8px">
```

Reply:
whoops my bad i'll change it to a `p` tag instead